packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -232,15 +389,167 @@ const PreviousDateRangePickerImpl = ({
 						</Menu.Item>
 					</>
 				) : (
-					<Menu.Item
-						style={{ padding: 0 }}
-						onClick={(e) => {
-							e.preventDefault()
-							e.stopPropagation()
-						}}
-					>
-						<DatePicker />
-					</Menu.Item>
+					<>
+						<Box
+							borderBottom={'divider'}
+							pb={'4'}
+							px={'4'}
+							display={'flex'}
+							gap={'4'}
+						>
+							<Box style={{ width: 116 }}>
+								<Box
+									border={'secondary'}
+									py={'8'}
+									borderBottom={
+										startTimeIsValid ? 'secondary' : 'none'
+									}
+									borderTopLeftRadius={'6'}
+									borderTopRightRadius={'6'}
+									borderBottomLeftRadius={
+										showingTime ? undefined : '6'
+									}
+									borderBottomRightRadius={
+										showingTime ? undefined : '6'
+									}
+									pl={'6'}
+								>
+									<Text size="small">
+										{selectedDates[0]
+											? formatDisplayedDate(
+													selectedDates[0],
+											  )
+											: 'Start Date'}
+									</Text>
+								</Box>
+								{showingTime ? (
+									<Box
+										border={
+											startTimeIsValid
+												? 'secondary'
+												: 'error'
+										}
+										borderTop={
+											startTimeIsValid ? 'none' : 'error'
+										}
+										borderBottomLeftRadius={'6'}
+										borderBottomRightRadius={'6'}
+									>
+										<Form.Input
```

Comment:
hey @betty-alagwu , apologies for all the back-and-forth. looks like there is a weird behavior now that bumps the input cursor to the end of the line whenever the time value is changed. here's an example of the behavior - https://www.loom.com/share/3c56e1726f784e26a4c0985d4914f53f

Reply:
Thank you for the feedback @Vadman97 , I will implement the changes today and I am happy to making any additional modifications.

Reply:
Hey @Vadman97 I've implemented some changes based on the bug you mentioned earlier.
Here's a video of how it works.

https://user-images.githubusercontent.com/94234459/234977177-ff92fc01-e2a3-4071-87fd-0a8902d9f987.mp4

Reply:
thanks @betty-alagwu ! this works great now